src/renderers/server-render.jsx

```diff
@@ -9,11 +9,10 @@ import ReactDOMServer from 'react-dom/server';
 import Helmet from 'react-helmet';
 import { API_ROUTE_PATH } from '../plugins/api-proxy'; // mwp-api-proxy
 import { Forbidden, NotFound, Redirect, SERVER_RENDER } from '../router'; // mwp-router
+import { getServerCreateStore } from '../store/server'; // mwp-store
+import Dom from '../render/components/Dom'; // mmwp-render/components/Dom
```

Comment:
`mmwp`

Do these comments have any programmatic significance now, or just planning for where we want them in the future?